source/extensions/filters/listener/tls_inspector/config.cc

```diff
@@ -27,7 +28,7 @@ class TlsInspectorConfigFactory : public Server::Configuration::NamedListenerFil
   }
 
   ProtobufTypes::MessagePtr createEmptyConfigProto() override {
-    return std::make_unique<Envoy::ProtobufWkt::Empty>();
+    return std::make_unique<envoy::config::filter::listener::tls_inspector::v2::TlsInspector>();
```

Comment:
So, do we know that if a `typed_config()` comes along with `Empty`, it will be correctly ingested? I assume some existing tests must cover this, but it would be good to see an example unit test that has both an `Empty` and empty typed message and verifies the same result.

Reply:
Yeah, it's worth testing but from code inspection it looks like it's going to kick-in JSON conversion from Empty.

Reply:
Added a unit test to validate Empty to empty proto conversion.